Ignore stderr (in case the file does not exists)
For upstream
Index: oar/setup/common.sh.in
===================================================================
--- oar.orig/setup/common.sh.in	2013-09-19 11:58:10.389755813 +0200
+++ oar/setup/common.sh.in	2013-09-19 15:11:18.878614657 +0200
@@ -85,11 +85,11 @@
 
 bash_oar
 EOF
-    if ! grep -q "${OARHOMEDIR}/.bash_oar" ${OARHOMEDIR}/.bash_profile; then
+    if ! grep -q "${OARHOMEDIR}/.bash_oar" ${OARHOMEDIR}/.bash_profile 2> /dev/null; then
         echo '' >> ${OARHOMEDIR}/.bash_profile
         echo "[ -f ${OARHOMEDIR}/.bash_oar ] && . ${OARHOMEDIR}/.bash_oar" >> ${OARHOMEDIR}/.bash_profile
     fi
-    if ! grep -q "${OARHOMEDIR}/.bash_oar" ${OARHOMEDIR}/.bashrc; then
+    if ! grep -q "${OARHOMEDIR}/.bash_oar" ${OARHOMEDIR}/.bashrc 2> /dev/null; then
         echo '' >> ${OARHOMEDIR}/.bashrc
         echo "[ -f ${OARHOMEDIR}/.bash_oar ] && . ${OARHOMEDIR}/.bash_oar" >> ${OARHOMEDIR}/.bashrc
     fi
